tests/__init__.py

```diff
@@ -183,7 +183,7 @@
         "double_repeat_example.krn",
         "fine_with_repeat.krn",
         "tuple_durations.krn",
-        "voice_dublifications.krn",
+        "voice_dublications.krn",
```

Comment:
another typo! It's "duplication"

Reply:
Addressed on commit [c1e27f7](https://github.com/CPJKU/partitura/pull/344/commits/c1e27f7553d9c9a636f4b74c2e7445d40e38281e)